src/engine/config.cpp

```diff
@@ -23,11 +23,84 @@
 #include <fmt/printf.h>
 
 #ifdef _WIN32
+#include "winStuff.h"
 #define CONFIG_FILE "\\furnace.cfg"
 #else
+#ifdef __HAIKU__
+#include <support/SupportDefs.h>
+#include <storage/FindDirectory.h>
+#endif
+#include <unistd.h>
+#include <pwd.h>
+#include <sys/stat.h>
 #define CONFIG_FILE "/furnace.cfg"
 #endif
 
+void DivEngine::initConfDir() {
+#ifdef _WIN32
+  // maybe move this function in here instead?
+  configPath=getWinConfigPath();
+#elif defined (IS_MOBILE)
+  configPath=SDL_GetPrefPath();
+#else
+#  ifdef __HAIKU__
```

Comment:
#ifdef (no indentation)

Reply:
Removed indentation of added pre-processor lines, done.